documentation/managing_modules.md

```diff
@@ -96,18 +95,72 @@ bolt module install --force
 Install-BoltModule -Force
 ```
 
+## Specify modules in a Bolt project
+
```

Comment:
```suggestion
In most cases, you can use the Bolt command line to install a new module. 
However, there are times when you might want to manually specify a module
instead of using the Bolt command line. For example, you might 
want to add a git module, which is not supported through the command 
line, or you might want to add a large number of modules to your project 
at once and you don't want to string together multiple commands.
```
This is just to provide a little context between using `add` or manually specifying